lib/track.ts

```diff
@@ -116,7 +116,7 @@ export class TrackClient {
     }
 
     return this.request.put(`${this.trackRoot}/customers/${encodeURIComponent(customerId)}/devices`, {
-      device: { id: device_id, platform, ...data },
+      device: { id: device_id, platform, attributes: { ...data } },
```

Comment:
While you're here @xanderberkein, let's add support for `last_used` here:

```suggestion
      let { last_used, ...attributes } = data;
      device: { id: device_id, platform, last_used, attributes },
```

Reply:
@mike-engel added in my latest commit